src/Effects/Effects.Android/Effects/Entry/EntryCapitalizeKeyboard.cs

```diff
@@ -6,28 +6,28 @@
 using Android.Runtime;
 using System.Linq;
 
-[assembly: ExportEffect(typeof(CapitalizeKeyboardEffect), nameof(CapitalizeKeyboardEffect))]
+[assembly: ExportEffect(typeof(EntryCapitalizeKeyboard), nameof(EntryCapitalizeKeyboard))]
 
```

Comment:
Do we want a newline here? I'd remove it and put attribute right on top of namespace.